### ISSUE TITLE
New release for cocoapods - Swift 5 & Eureka 5.0.0

### ISSUE THREAD

**Issue by ceyey4e2q** [EurekaCommunity/SuggestionRow#40]
Hi!

Is it possible to release a new version on cocoapods that works with Swift 5 + Eureka 5.0.0?

I can't update to Eureka 5.0.0 with 'pod outdated' + 'pod update' because SuggestionRow isn't compatible yet with this new version.

Error message:

```bash
[!] CocoaPods could not find compatible versions for pod "Eureka":
  In snapshot (Podfile.lock):
    Eureka (= 5.0.0, ~> 5.0)

  In Podfile:
    Eureka (~> 5.0)

    SuggestionRow (~> 2.2) was resolved to 2.2.0, which depends on
      Eureka (~> 4.0)
```

Thanks!!

**Comment by macfurin** [comment]
Same here, I would like to use Eureka 5.0

**Comment by mats-claassen** [comment]
Released v3.0.0
